storage/container.go

```diff
@@ -65,17 +125,38 @@ func (c *StorageClient) CreateContainer(input *CreateContainerInput) (*Container
 
 	// There are default values for these that we don't want to zero out if Read and Write ACLs are not set.
 	if len(input.ReadACLs) > 0 {
-		headers["X-Container-Read"] = strings.Join(input.ReadACLs, ",")
+		headers[hContainerRead] = strings.Join(input.ReadACLs, ",")
 	}
 	if len(input.WriteACLs) > 0 {
-		headers["X-Container-Write"] = strings.Join(input.WriteACLs, ",")
+		headers[hContainerWrite] = strings.Join(input.WriteACLs, ",")
 	}
 
-	headers["X-Container-Meta-Temp-URL-Key"] = input.PrimaryKey
-	headers["X-Container-Meta-Temp-URL-Key-2"] = input.SecondaryKey
-	headers["X-Container-Meta-Access-Control-Allow-Origin"] = strings.Join(input.AllowedOrigins, " ")
-	headers["X-Container-Meta-Access-Control-Expose-Headers"] = strings.Join(input.ExposedHeaders, " ")
-	headers["X-Container-Meta-Access-Control-Max-Age"] = strconv.Itoa(input.MaxAge)
+	headers[hTempURLKey] = input.PrimaryKey
+	headers[hTempURLKey2] = input.SecondaryKey
+	headers[hAccessControlAllowOrigin] = strings.Join(input.AllowedOrigins, " ")
+	headers[hAccessControlExposeHeaders] = strings.Join(input.ExposedHeaders, " ")
+	// headers[hPolicyGeoreplication] = strings.Join(input.GeoreplicationPolicy, " ")
```

Comment:
Is there a reason this is commented out?

Reply:
The Georeplication Policy header set on the container, but not covered in the API docs, so commented it out for now till its clear how this attribute should be used.

Reply:
Perfect